engines/batch-ensembler/entrypoint.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```

Comment:
What's the difference between this `entrypoint.sh` and endtrypoint available in the base spark docker image?

Reply:
Never knew this existed, let me try this out.

Reply:
The base entrypoint works, I'll remove this script.